x/reports/module.go

```diff
@@ -151,29 +152,45 @@ func (am AppModule) EndBlock(_ sdk.Context, _ abci.RequestEndBlock) []abci.Valid
 //____________________________________________________________________________
 
 // AppModuleSimulation defines the module simulation functions used by the posts module.
-type AppModuleSimulation struct{}
+type AppModuleSimulation struct {
```

Comment:
All the changes here should be reverted. No need to declare a new `AppModuleSimulation` type when the `AppModule` already implements this